website_sale_qty/wizards/website_config_settings.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 LasLabs Inc.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html) -->
+
+<odoo>
+    <record id="view_website_config_settings" model="ir.ui.view">
+        <field name="name">Website Settings - Implicit Price Tier</field>
+        <field name="model">website.config.settings</field>
+        <field name="inherit_id" ref="website.view_website_config_settings"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='delivery']" position="inside">
```

Comment:
@obulkin - I figured out what this is, and it's most certainly an issue (not sure why tests aren't failing). 

You're targeting `website.view_website_config_settings` as the inherit here, but the `delivery` group is introduced in [`website.website_config_settings_view_form`](https://github.com/odoo/odoo/blob/fc2e80cb4bcc450762c7ac5cb82a3e2d88062b38/addons/website_sale/views/res_config_view.xml#L3)